app/models/taxon_presenter.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
+
+var _ = require('lodash');
+var urlHelper = require('url');
+
+var Taxon = require('./taxon.js');
+var BreadcrumbMaker = require('../../lib/js/breadcrumb_maker.js');
+
+class TaxonPresenter {
+  constructor (taxonParam, taxonomyData) {
+    this.taxonomyData = taxonomyData;
+    this.basePath = "/alpha-taxonomy/" + taxonParam;
+
+    this.build();
+  }
+
+  build () {
+    this.buildTaxon();
+    this.buildBreadcrumb();
+    this.buildParent();
+    this.buildChildren();
+    this.buildContent();
+    this.checkIfPenultimate();
+  }
+
+  buildTaxon () {
+    this.taxon = Taxon.fromMetadata(this.basePath, this.taxonomyData);
+    this.title = this.taxon.title;
+    this.description = this.taxon.description;
+  }
+
+  buildBreadcrumb () {
+    var breadcrumbMaker = new BreadcrumbMaker(this.taxonomyData);
+    this.breadcrumb =  breadcrumbMaker.getBreadcrumbForTaxon([this.basePath]);
+  }
+
+  buildParent () {
+    this.parent = this.breadcrumb[this.breadcrumb.length - 1];
+  }
+
+  buildChildren () {
+    this.children = this.taxon.atozChildren();
+    for (var i = 0; i < this.children.length; i += 1) {
+      this.children[i].guidance = this.children[i].filterByHeading('guidance');
+    }
+  }
+
+  buildContent () {
+    this.content = {}
+    this.content.guidance = this.taxon.filterByHeading('guidance');
+  }
+
+  checkIfPenultimate () {
+    var findGrandchild = _.find(this.children, function (childTaxon) {
+      return childTaxon.children.length > 0;
+    });
```

Comment:
We could perhaps remove the `lodash` dependency and use the array methods here together with the new function syntax as well. What do you think of something like:

```
checkIfPenultimate () {
  var ifPenultimate = (childTaxon) => childTaxon.children.length > 0;
  var findGrandchild = this.children.find(ifPenultimate);
```